test/transform/test_graph_2_simplicial_complex.py

```diff
@@ -0,0 +1,169 @@
+"""Test graph to simplicial complex transformation."""
```

Comment:
The name of this file should be test_[name of file being tested]

i.e test_graph_to_simplicial_complex

Reply:
I saw that the conversion functions themselves use "_2_". Could you rename this to "_to_" everywhere, for consistency, as well?

Many thanks.